### PR TITLE
Fix all time no results

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- [Fixes this Sentry error](https://sentry.io/organizations/posthog/issues/1648291886/?project=1899813&referrer=slack)

## Checklist
- [x] All querysets/queries filter by Team (if applicable)
- [x] Backend tests (if applicable)

### CODE REVIEW

**Review by EDsCODE — COMMENTED** [review]
All time is returning nothing for me (on demo instance). 
Extra info that might help:
I printed values to take a quick took and `aggregates` returns the right values right before entering `_group_events_to_date`. I think there's a mismatch on datetime from python and timestamps from database. It also may have grouping by exact timestamp instead of a range. The date objects in aggregates before entering `_group_events_to_date` are datetimes on midnight of day

**Comment by timgl** [comment]
@EDsCODE Fixed! Thanks for digging

File: posthog/api/action.py [hunk 1]
```diff
@@ -363,6 +365,13 @@ def trends(self, request: request.Request, *args: Any, **kwargs: Any) -> Respons
             # If no filters, automatically grab all actions and show those instead
             filter.entities = [Entity({'id': action.id, 'name': action.name, 'type': TREND_FILTER_TYPE_ACTIONS}) for action in actions]
 
+        if not filter.date_from:
+            filter._date_from = Event.objects.filter(team=team)\
+                .order_by('timestamp')[0]\
+                .timestamp\
+                .replace(hour=0, minute=0, second=0, microsecond=0)\
+                .isoformat()
+
```
Comment (EDsCODE) [inline posthog/api/action.py:374]: Handle the case where `if not filter.date_to` here and setting `filter._date_to=now().isoformat()`

File: posthog/api/action.py [hunk 2]
```diff
@@ -238,7 +238,7 @@ def _aggregate_by_interval(self, filtered_events: QuerySet, entity: Entity, filt
         aggregates = self._process_math(aggregates, entity)
 
         dates_filled = self._group_events_to_date(
-            date_from=filter.date_from if filter.date_from else pd.Timestamp(aggregates[0][interval]),
+            date_from=filter.date_from,
             date_to=filter.date_to if filter.date_to else now(),
```
Comment (EDsCODE) [inline posthog/api/action.py:242]: And remove this now()

**Review by EDsCODE — CHANGES_REQUESTED** [review]
All time looks good. Stickiness isn't working because it requires `date_to` and `date_from` but the null case for `date_to` is being handled inside `_aggregate_by_interval` and stickiness doesn't have any handling

**Comment by timgl** [comment]
Thanks for debugging this! Added a test case too and should be fixed now

**Review by EDsCODE — APPROVED** [review]

**Pull request merged**